### DISCUSSION TITLE
How about release a new version of circus

### DISCUSSION THREAD

**Discussion by tinylambda** [circus-tent/circus#1187] [General]
The last release date is Sep 19, 2020, 0.17.1

It's time to release a new version 0.17.2 .

@k4nar Can you release a new version and put it onto pypi ? thanks.

**Comment by k4nar** [comment]
Good idea, I'll try to find some time to do it :+1: .

**Reply by tripougnif** [reply]
That would be much appreciated. I'm on Archlinux and I'm too lazy to setup a container with some other distro that is less aggressive about upgrades. So I'm looking forward to a new release mainly because of #1170.

**Comment by sphuber** [comment]
Following the mantra "better late than never", I have just released `v0.17.2`: https://pypi.org/project/circus/0.17.2/
